<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VigenereCipherMainWindow</class>
 <widget class="QMainWindow" name="VigenereCipherMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VigenereCipherMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QFrame" name="frKey">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLineEdit" name="leKey"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lKeyHeader">
         <property name="text">
          <string>Ключ</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pbRandomKey">
         <property name="text">
          <string>Сгенерировать случайный ключ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frOriginalData">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lOriginalDataHeader">
           <property name="text">
            <string>Исходные данные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="teOriginalData"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbEncryptData">
           <property name="text">
            <string>Зашифровать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDecryptData">
           <property name="text">
            <string>Расшифровать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frDecryptedData">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lDecryptedDataHeader">
           <property name="text">
            <string>Преобразованные данные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="teDecryptedData">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbCopyInClipboard">
           <property name="text">
            <string>Скопировать в буфер обмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbClearForm">
      <property name="text">
       <string>Очистить форму</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
